sentence_transformers/evaluation/RerankingEvaluator.py

```diff
@@ -35,7 +35,7 @@ def __init__(
     ):
         self.samples = samples
         self.name = name
```

Comment:
```suggestion
        mrr_at_k: Optional[int] = None,
    ):
        self.samples = samples
        self.name = name
        if mrr_at_k is not None:
            logger.warning(f"The `mrr_at_k` parameter has been deprecated; please use `at_k={mrr_at_k}` instead.")
            at_k = mrr_at_k
        else:
            at_k = at_k
```

Reply:
Thank you for the fast feedback and your suggestions! This completely makes sense.